dev/NavigationView/NavigationView.xaml

```diff
@@ -285,6 +285,40 @@
                                                         <Setter Property="Padding" Value="{ThemeResource TopNavigationViewOverflowMenuPadding}" />
                                                         <!-- Set negative top margin to make the flyout align exactly with the button -->
                                                         <Setter Property="Margin" Value="0,-4,0,0" />
+                                                        <Setter Property="ScrollViewer.HorizontalScrollMode" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.VerticalScrollMode" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.VerticalScrollBarVisibility" Value="Auto" />
+                                                        <Setter Property="ScrollViewer.ZoomMode" Value="Disabled" />
+                                                        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
+                                                        <Setter Property="Template">
+                                                            <Setter.Value>
+                                                                <ControlTemplate TargetType="FlyoutPresenter">
+                                                                    <ScrollViewer
+                                                                        x:Name="ScrollViewer"
+                                                                        ZoomMode="{TemplateBinding ScrollViewer.ZoomMode}"
+                                                                        HorizontalScrollMode="{TemplateBinding ScrollViewer.HorizontalScrollMode}"
+                                                                        HorizontalScrollBarVisibility="{TemplateBinding ScrollViewer.HorizontalScrollBarVisibility}"
+                                                                        VerticalScrollMode="{TemplateBinding ScrollViewer.VerticalScrollMode}"
+                                                                        VerticalScrollBarVisibility="{TemplateBinding ScrollViewer.VerticalScrollBarVisibility}"
```

Comment:
How are these values customizable? At first glance it seems like it would be pretty difficult, in that case are the template bindings needed?

Reply:
Those template bindings have just been copied over from the default FlyoutPresenter control template. I think it makes sense to replace these bindings with the actual values like `HorizontalScrollBarVisibility.Auto`.

Reply:
Hmm, I want to say that we should remove them, because while template bindings are cheap they aren't free and there are potentially quite a few of thes flyouts. However Even though NavView isn't changing any of these flyout properties I'm not confident that flyout itself, or perhaps button doesn't set some of them, so I'm not sure it is safe to remove them. @RanjeshJ for this thoughts, but I think leaving them here is safer and probably fine.

---
In reply to: [468170773](https://github.com/microsoft/microsoft-ui-xaml/pull/3082#discussion_r468170773) [](ancestors = 468170773)

Reply:
I think it is safer to leave them there.